WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -703,26 +705,20 @@ protected void showErrorDialogWithCloseButton(String message) {
     }
 
     protected void startCropActivity(Uri uri) {
-        final Context context = getActivity();
-
-        if (context != null) {
-            UCrop.Options options = new UCrop.Options();
-            options.setShowCropGrid(false);
-            options.setStatusBarColor(ContextExtensionsKt.getColorFromAttribute(
-                    context, android.R.attr.statusBarColor
-            ));
-            options.setToolbarColor(ContextExtensionsKt.getColorFromAttribute(context, R.attr.wpColorAppBar));
-            options.setToolbarWidgetColor(ContextExtensionsKt.getColorFromAttribute(
-                    context, R.attr.colorOnPrimarySurface
-            ));
-            options.setAllowedGestures(UCropActivity.SCALE, UCropActivity.NONE, UCropActivity.NONE);
-            options.setHideBottomControls(true);
-
-            UCrop.of(uri, Uri.fromFile(new File(context.getCacheDir(), "cropped.jpg")))
-                 .withAspectRatio(1, 1)
-                 .withOptions(options)
-                 .start(getActivity(), this);
-        }
+        final Context context = new ContextThemeWrapper(getActivity(), R.style.WordPress_NoActionBar);
```

Comment:
We're removing the getActivity() != null check. Is it possible it might cause a problem?

Reply:
Good catch! `ContextThemeWrapper` will be non-null even if the base `context` is, which could cause problems indeed. 

I've made some changes to check if the base `context` is null before wrapping it with the `ContextThemeWrapper`.